src/features/emails/hooks/useLinks.ts

```diff
@@ -0,0 +1,30 @@
+import { IFuture } from 'core/caching/futures';
+import { loadListIfNecessary } from 'core/caching/cacheUtils';
+import { ZetkinLink } from 'utils/types/zetkin';
+import { linksLoad, linksLoaded } from '../store';
+import { useApiClient, useAppDispatch, useAppSelector } from 'core/hooks';
+
+export default function useLinks(
```

Comment:
This hook loads links for a single email specifically, but stores them in such a way in the store that it's not clear that they belong only to a single email. This means that if links are loaded for one `emailId`, it will not load again when this hook is used with another `emailId` (because `linkList.loaded` is truthy).